inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/instrumentation/context/ContextManager.java

```diff
@@ -22,6 +24,8 @@
      */
     public static final String BEAN_NAME = "contextManager";
 
+    private static final boolean IS_OPEN_CENSUS_ON_BOOTSTRAP = Tags.class.getClassLoader() == null;
```

Comment:
although this is quite a nice solution, would this also represent a problem in the OSGi env, as you would try to load this class when our agent is initializing.. maybe simply read the infomation from the config in the constructor `pushOcToBootstarp`?